Automattic/wp-calypso
# Added "Finish store setup" card to My Home

#### Changes proposed in this Pull Request

* This PR is part of a bigger change related to the WC-Admin issue [554](https://github.com/Automattic/wc-calypso-bridge/issues/554).

* It focuses on only the visual changes.

* This PR adds the "Finish store setup" card to `My Home`.

![screenshot-calypso localhost_3000-2020 09 07-10_55_45](https://user-images.githubusercontent.com/1314156/92411318-bc141600-f11d-11ea-8447-109fdeede5d7.png)


#### Testing instructions



In order to test this PR, [these changes](https://code.a8c.com/D49062) should be approved in `Phabricator` first.

* Choose one page and go to `My Home`, adding the params`?dev=true&view=VIEW_SITE_SETUP_ECOMMERCE` to the URL  (it will look like `http://calypso.localhost:3000/home/[my_site]?dev=true&view=VIEW_SITE_SETUP_ECOMMERCE`).
* Verify that the `Finish store setup` card is visible.
* Verify that the `Quick link` named `Finish store setup` is visible too.
* The button `Finish store setup` should be pointing to the site's store.
* Verify that the description text and the image in the card are correct.

Fixes partially [554](https://github.com/Automattic/wc-calypso-bridge/issues/554)
